### PR TITLE
Pop up window for the projects

### PR DESCRIPTION
**Pull request by Aliahmad-karimi**
### Pop up 

- Declare for each project a pop-up window for mobile and desktop
- Declare the function that get the information from object array to the HTML elements
-

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -736,26 +736,27 @@ footer .submit:active {
...
 .popUp .popHead {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
-  gap: 12px;
   width: 311px;
-  height: 84px;
+  height: 100px;
+  margin-bottom: 12px;
 }
```
Comment (ShahierNashaat) [inline style.css]: - [ ] Kindly be noticed that you should make a scroll for the pop window project in the desktop size, to make the user see the description, the technologies, and the links for the source and live demo.

Screenshot from 2022-07-27 00-59-22

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Aliahmad-karimi) [reply style.css]: Can u give me more information about it, sir

Comment (Aliahmad-karimi) [reply style.css]: @ShahierNashaat Dear please give me a few details about the changes u want from me
I am still confused by your request.

Comment (ShahierNashaat) [reply style.css]: I can't scroll down to see the description of the project please fix that

**Review by omarsalem7 — APPROVED** [review]
Hi Team 😊, 

# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️
- Descriptive pull request ✔️

Cheers and Happy coding!👏👏👏

## Optional suggestions

- [ ] [OPTIONAL] It will be better to add a blur effect to the background behind the popup window to be consistent with the Figma template.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Aliahmad-karimi** [comment]
@ShahierNashaat dear I will try to do that one, but before that, I need to complete the week. I will try to do that one.

**Pull request merged**